doc/developer/interface/custom-login/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+---
+
+
+Custom Login Page
+==========================================================
+
+The default landing / login page for IRIDA can be replaced by an institution without modifying the existing one.
+
+Create a html file called `login.html` on your files system.  The default path IRIDA will check is `/etc/irida/templates`, but can be configured in the configuration file `src/main/resources/configuration.properties` by updating the `ui.templates` property,
+
+```properties
+# Overwritten UI templates
+ui.templates=/etc/irida/templates/
+```
+
+### Use IRIDA base login form.
+
+IRIDA uses a React component as its login form and can be easily rendered on and DOM node.  The basic layout can be found in the code snippet below:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" th:href="@{/dist/css/login.bundle.css}" />
+    <script th:inline="javascript">
+      window.PAGE = {
+        BASE_URL: /*[[@{"/"}]]*/ "/"
+      };
+    </script>
+  </head>
+  <body>
+    <div id="login-root"></div>
+    <script th:src="@{/dist/js/login.bundle.js}"></script>
```

Comment:
Are they able to add their own JS/CSS/image resources to be displayed in this page as well?  If so how are they configured and served?

Reply:
Added a small not in 3a0eb660bc720bca96ec3c70e1f70e5c5da08c17.  This is something we can explore more later.  Perhaps they could use their own build system to store script in say `/etc/irida/dist/` or something.  This might actually also work for plugin pipelines.